### PR TITLE
Update gw-auto-complete to check user agent and adjust attribute automatically

### PR DESCRIPTION
**Pull request by scyt**

### CODE REVIEW

**Review by spivurno — APPROVED** [review]
Made a few changes. Could you confirm I didn't introduce any fatal errors? 😅

If not, approved!

File: gravity-forms/gw-disable-autocomplete.php [hunk 1]
```diff
@@ -20,20 +20,22 @@
...
-	} elseif ( strpos( $user_agent, 'Chrome' ) ) {
-		return 'Chrome';
-	} elseif ( strpos( $user_agent, 'Safari' ) ) {
-		return 'Safari';
-	} elseif ( strpos( $user_agent, 'Firefox' ) ) {
-		return 'Firefox';
-	} elseif ( strpos( $user_agent, 'MSIE' ) || strpos( $user_agent, 'Trident/7' ) ) {
-		return 'Internet Explorer';
-	}
+if ( ! function_exists( 'gw_get_browser_name' ) {
```
Comment (scyt) [inline gravity-forms/gw-disable-autocomplete.php:23]: This if statement is missing a closing parenthesis.

**Review by scyt — COMMENTED** [review]
There's a missing parenthesis on line 23. Adding in next commit.

**Review by spivurno — APPROVED** [review]
Good catch. Looks good!

**Pull request merged**